packages/api/lib/testrun.js

```diff
@@ -20,11 +20,11 @@ class TestCaseRun {
     this.number = number;
     this.name = name;
     this.encodedName = firebaseEncode(this.name);
-    this.failureMessage = 'TODO ERROR MESSAGE, (e.g. stackoverflow error line 13)';
+    this.failureMessage = 'Successful';
```

Comment:
Nit: I find `failureMessage` being `Successful` a bit weird, should this be something like `statusMessage`.

I'm okay with avoiding the refactor and keeping `failureMessage` depending on the hassle ... I almost like this little quirk 😆